frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -83,13 +86,48 @@ async function deleteUserRoleAssignment(
 
         <div class="dashboard-background-layout">
             <NotificationStack />
-            <UserDataTable
-                :isApplicationSelected="isApplicationSelected"
-                :loading="LoadingState.isLoading.value"
-                :userRoleAssignments="userRoleAssignments || []"
-                :selectedApplicationDisplayText="selectedApplicationDisplayText"
-                @deleteUserRoleAssignment="deleteUserRoleAssignment"
-            />
+            <TablePlaceholder v-if="!isApplicationSelected" />
+            <TabView
+                v-else
+                :pt="{
+                    root: {
+                    style: 'margin-top: 1.5rem',
+                    },
+                    nav: {
+                    style: 'margin: 0.112rem 2.5rem 0;',
+                    },
+                    panelContainer: {
```

Comment:
the format here doesn't look right, you might want to manually correct it. Also you might not need to style change for panelContainer anymore.

Reply:
These styles are needed, they were on the common repo but since it was only for fam I migrated them to local

Reply:
once you cleanup the style change as mention above and in the common repo, you just need the following, the rest can be removed. So when we do style change, we have to understand the logic as well, and the relationship between its parent component and sibling component. 
<img width="571" alt="Screen Shot 2023-12-14 at 2 29 52 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/7b454f4b-979b-4f34-abba-e4488e7b8ee0">